apps/supernova/docker/Dockerfile

```diff
@@ -7,64 +7,17 @@ LABEL org.opencontainers.image.source=https://github.com/cloudoperators/juno/tre
 LABEL org.opencontainers.image.description="This image contains a standalone runnable version of the Supernova alternative Alertmanager UI. For configuration options see the README here: https://github.com/cloudoperators/juno/tree/main/apps/supernova/docker"
 LABEL org.opencontainers.image.licenses=Apache-2.0
 
-# we need jq to read and manipulate package.json and upgrade the system
-RUN apk add --no-cache jq \
-  && apk upgrade --no-cache --no-progress \
-  && apk del --no-cache --no-progress apk-tools alpine-keys
+WORKDIR /app  
 
-# create a directory where we can copy the source code
-RUN mkdir -p /tmp/src /app
-# copy the repository code to the container
-ADD . /tmp/src
-# app specific environment variables
-ENV PACKAGE_PATH="apps/supernova"
+ADD . .
 
-# install dependencies but from root directory of the project
-# build the project but from root directory of the project 
-# copy the build output to /app
-# copy favicon.ico to /app
-RUN \
-  cd /tmp/src && \
-  entry_file=$(jq -r '.module // .main' $PACKAGE_PATH/package.json) && \
-  echo $entry_file > /app/entry_file && \
-  dir=$(dirname $entry_file) && \
-  package_name=$(jq -r '.name' $PACKAGE_PATH/package.json) && \
-  echo "Entry file: $entry_file" && \
-  echo "Directory: $dir" && \
-  echo "Package name: $package_name" && \
-  npm install && \
-  npx turbo run build --filter $package_name && \
-  cp -r /tmp/src/$PACKAGE_PATH/$dir /app/ && \
-  cp $PACKAGE_PATH/public/favicon.ico /app/
-
-# create index.html  
-RUN \
-  entry_file=$(cat /app/entry_file) && \
-  entry_file_escaped=$(echo "$entry_file" | sed 's/\//\\\//g') && \
-  cp /tmp/src/$PACKAGE_PATH/docker/index.html /app/index.html && \
-  sed -i "s/{{entry_file}}/$entry_file_escaped/g" /app/index.html  
-
-# create appProps.js
-RUN cat <<EOF > /app/appProps.json
-{
-  "endpoint": "",
-  "theme": "theme-dark",
-  "embedded": false,
-  "silenceExcludedLabels": "",
-  "silenceTemplates": ""
-}
-EOF
-
-# clean up
-RUN rm /app/entry_file
+RUN npm install && npx turbo build:static --filter @cloudoperators/juno-app-supernova
 
 FROM nginx:alpine
+# we need to install jq, cause it is used in the entrypoint.sh
+RUN apk add --no-cache jq && apk upgrade --no-cache --no-progress 
 
-RUN apk add --no-cache jq \
-  && apk upgrade --no-cache --no-progress \
-  && apk del --no-cache --no-progress apk-tools alpine-keys
-
-COPY --from=build /app /usr/share/nginx/html
+WORKDIR /usr/share/nginx/html
 
 ENV THEME=""
```

Comment:
Can we automate the creation of environment variables based on the `appProps.template.json` file? As mentioned earlier, we could use `appProps.template.json` to generate `appProps.json`, and it could also be leveraged for this step.

Something like this:
```
# Loop through the JSON keys and create ENV variables
for key in $(jq -r 'keys_unsorted[]' "$json_file"); do
  value=$(jq -r ".\"$key\"" "$json_file")
  echo "ENV $key=\"$value\"" >> "$env_file"
done
```